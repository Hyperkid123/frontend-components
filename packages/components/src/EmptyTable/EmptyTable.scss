@import '~@redhat-cloud-services/frontend-components-utilities/styles/_all.scss';

.ins-c-table__empty {
  background: var(--pf-v5-global--Color--light-100);
  border-bottom: 1px solid var(--pf-v5-global--BorderColor--light-200);
  @include rem('padding', 20px);
  &.is-centered {
    display: flex;
    justify-content: center;
  }
}
