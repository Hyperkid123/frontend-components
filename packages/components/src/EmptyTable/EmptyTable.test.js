import React from 'react';
import EmptyTable from './EmptyTable';
import { render } from '@testing-library/react';

describe('EmptyTable component', () => {
  it('should render', () => {
    const { container } = render(<EmptyTable>Some</EmptyTable>);
    expect(container).toMatchSnapshot();
  });

  it('should render with centered children', () => {
    const { container } = render(<EmptyTable centered>Centered</EmptyTable>);
    expect(container).toMatchSnapshot();
  });
});
