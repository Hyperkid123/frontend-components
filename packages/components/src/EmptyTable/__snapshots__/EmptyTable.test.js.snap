// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTable component should render 1`] = `
<div>
  <div
    class="ins-c-table__empty"
  >
     
    Some
  </div>
</div>
`;

exports[`EmptyTable component should render with centered children 1`] = `
<div>
  <div
    class="ins-c-table__empty is-centered"
  >
     
    Centered
  </div>
</div>
`;
